addons/im_livechat/static/src/core/web/thread_actions.js

```diff
@@ -6,7 +6,9 @@ import { LivechatChannelInfoList } from "@im_livechat/core/web/livechat_channel_
 registerThreadAction("livechat-info", {
     actionPanelComponent: LivechatChannelInfoList,
     condition: ({ owner, thread }) =>
-        thread?.channel_type === "livechat" && !owner.isDiscussSidebarChannelActions,
+        thread?.channel_type === "livechat" &&
+        thread.livechatVisitorMember?.notEq(thread.self_member_id) &&
```

Comment:
`livechatVisitorMember` is not a good field to use (assumes there is only one, uses heuristics), should use `livechat_member_type` instead